src/sentry/sentry_metrics/client/snuba.py

```diff
@@ -41,6 +48,113 @@ class SnubaMetricsBackend(GenericMetricsBackend):
     metric into Clickhouse.
     """
 
+    def _send_buckets(self, buckets, entity):
+        if entity.startswith("generic_"):
+            codec = sentry_kafka_schemas.get_codec("snuba-generic-metrics")
+        else:
+            codec = sentry_kafka_schemas.get_codec("snuba-metrics")
+
+        for bucket in buckets:
+            codec.validate(bucket)
+
+        snuba_endpoint = "/tests/entities/{entity}/insert"
```

Comment:
This endpoint is not available in production.
The request would generate a 404.

And we cannot not make it available. See the overall PR review comment.